plugin-server/src/worker/ingestion/persons/repositories/postgres-person-repository.test.ts

```diff
@@ -684,41 +688,44 @@ describe('PostgresPersonRepository', () => {
                 },
             })
 
-            const size = await repository.personPropertiesSize(team.id, 'test-distinct')
+            const size = await repository.personPropertiesSize(person.id)
 
             expect(size).toBeGreaterThan(0)
             expect(typeof size).toBe('number')
         })
 
         it('should return 0 for non-existent person', async () => {
-            const team = await getFirstTeam(hub)
-            const size = await repository.personPropertiesSize(team.id, 'non-existent-distinct')
+            const fakePersonId = new UUIDT().toString()
+            const size = await repository.personPropertiesSize(fakePersonId)
 
             expect(size).toBe(0)
         })
 
-        it('should handle different team IDs correctly', async () => {
+        it('should handle different persons correctly', async () => {
             const team1 = await getFirstTeam(hub)
             const team2Id = await createTeam(hub.db.postgres, team1.organization_id)
 
             // Create person in team 1
-            await createTestPerson(team1.id, 'shared-distinct', { name: 'Team 1 Person' })
+            const person1 = await createTestPerson(team1.id, 'shared-distinct', { name: 'Team 1 Person' })
 
-            // Check size in team 1
-            const size1 = await repository.personPropertiesSize(team1.id, 'shared-distinct')
+            // Create person in team 2
+            const person2 = await createTestPerson(team2Id, 'different-distinct', { name: 'Team 2 Person' })
+
+            // Check size for person 1
+            const size1 = await repository.personPropertiesSize(person1.id)
             expect(size1).toBeGreaterThan(0)
 
-            // Check size in team 2 (should be 0 since person doesn't exist there)
-            const size2 = await repository.personPropertiesSize(team2Id, 'shared-distinct')
-            expect(size2).toBe(0)
+            // Check size for person 2
+            const size2 = await repository.personPropertiesSize(person2.id)
+            expect(size2).toBeGreaterThan(0)
```

Comment:
nit: would also be useful to set different properties for those 2 persons and and check that the sizes differ.